src/tempo_cluster.py

```diff
@@ -161,10 +163,10 @@ def dump(self, databag: Optional[MutableMapping] = None, clear: bool = True):
                 databag = {}
 
             if self._NEST_UNDER:
-                databag[self._NEST_UNDER] = self.json(by_alias=True, exclude_defaults=True)
+                databag[self._NEST_UNDER] = self.json(by_alias=True, exclude_none=True)
```

Comment:
Do we still need `_NEST_UNDER`?
Iirc, it was introduced into traefik because at the time the old pydantic had some limitations.